src/command/angi.command.ts

```
@@ -41,35 +55,61 @@ export class AnGiCommand extends CommandMessage {
         const lines: string[] = [];
         
         // Main suggestion
-        lines.push(`🍽️ **Gợi ý hôm nay: ${result.picked.name}**`);
-        lines.push(`🏙️ Tỉnh/Thành: ${result.picked.province}`);
-        lines.push(`🗺️ Miền: ${result.picked.region}`);
-        lines.push(`📋 Phân loại: ${result.picked.category}`);
+        lines.push(formatMessage(ANGI_MESSAGES.INFO.DISH_DETAIL_ITEM, {
```

ESLint

[error] Replace `····lines.push(` with `lines.push(⏎······`

(prettier/prettier)